packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -598,4 +598,6 @@ export enum UsageFlags {
   JsonMergePatch = 1 << 4,
   // Input will also be set when MultipartFormData is set.
   MultipartFormData = 1 << 5,
+  // Used in spread.
+  Spread = 1 << 6,
```

Comment:
Maybe not relevant to this pr, just want to clarify UsageFlags, if a model is used in 
- patch operation for input
- another operation for spread input
- another operation for output

How many medels we will get from TCGC?

Reply:
two. one original model with path|outpu, one anonymous model with spread.